src/ethereum_test_vm/opcode.py

```diff
@@ -5097,7 +5097,7 @@ class Opcodes(Opcode, Enum):
     """
     !!! Note: This opcode is under development
 
-    EOFCREATE[initcontainer_index](value, salt, input_offset, input_size)
+    EOFCREATE[initcontainer_index] value, salt, input_offset, input_size
```

Comment:
```suggestion
    EOFCREATE[initcontainer_index] (value, salt, input_offset, input_size)
```
Preserves the text mostly as-is while inhibiting the link creation.